pkg/controller/linstorcontrollerset/linstorcontrollerset_controller.go

```diff
@@ -428,8 +393,24 @@ func (r *ReconcileLinstorControllerSet) reconcileControllerNodeWithControllers(p
 	return nil
 }
 
+func (r *ReconcileLinstorControllerSet) reconcileStatus(ctx context.Context, pcs *piraeusv1alpha1.LinstorControllerSet, errs []error) error {
+	log := logrus.WithFields(logrus.Fields{
+		"Name":      pcs.Name,
+		"Namespace": pcs.Namespace,
+	})
+	log.Debug("reconcile status")
+
+	log.Debug("reconcile error list")
```

Comment:
We don't really need 2 log messages directly one after the other. I would combine them.